integration-core/src/main/java/org/symphonyoss/integration/core/bootstrap/IntegrationBootstrapContext.java

```diff
@@ -245,6 +265,32 @@ protected void execute() {
     });
   }
 
+  public void logHealthStatus(Integration integration)
```

Comment:
Missing Javadoc. I'm also not sure if I understand the proposal of this method.

But I believe you should split this method into other two methods: logHealthCheck() and logIntegrationHealthCheck(Integration integration).